installer/manifests/keptn/charts/control-plane/values.yaml

```diff
@@ -86,6 +86,7 @@ apiService:
     repository: docker.io/keptn/api
     tag: ""
   config:
+    maxAuthEnabled: true
```

Comment:
Maybe it would give us a better structure if we had a separate block called `maxAuth` and then have `enabled`, `requestsPerSecond` and `requestsBurst` inside?